lib/createsend/utils.py

```diff
@@ -171,6 +171,6 @@ def createsend_url(self, url):
             if url.startswith("http"):
                 return url
             else:
-                return "https://api.createsend.com/api/v3.2/%s" % url
+                return "https://api.createsend.com/api/v3.3/%s" % url
```

Comment:
This is used for testing -- we verify that the endpoint that was hit is as expected (ie: expecting 3.3 now)